### DISCUSSION TITLE
Del on the end of the line

### DISCUSSION THREAD

**Discussion by adsick** [helix-editor/helix#10146] [General]
When I <kbd>gla</kbd> in normal mode and then <kbd>del</kbd> I expect the next line to be deleted if it has nothing but tabs in it - VSCode does it, Helix - not.

Basically I think that del should delete next line if it all it has is tabs and/or whitespaces.

**Comment by dead10ck** [comment]
I'm not sure about this. On one hand, this doesn't align with the select-act model. On the other hand, the practical effect of this behavior is just ending up with a bunch of whitespace at the end of the current line, which is hard to imagine anyone could want.
